Shared/Samples/Show device location using indoor positioning/ShowDeviceLocationUsingIndoorPositioningView.swift

```diff
@@ -0,0 +1,75 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import CoreLocation
+import SwiftUI
+
+struct ShowDeviceLocationUsingIndoorPositioningView: View {
+    /// The data model for the sample.
+    @StateObject private var model = Model()
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    var body: some View {
+        MapView(map: model.map)
+            .locationDisplay(model.locationDisplay)
+            .overlay(alignment: .top) {
+                HStack(alignment: .center, spacing: 10) {
+                    Text(model.labelTextLeading)
+                    Text(model.labelTextTrailing)
```

Comment:
The HUD text is a little difficult to read. Consider making the text darker and/or the background not as transparent. Even better, consider using a [material](https://developer.apple.com/documentation/swiftui/material/) for the background so that the text has a vibrancy effect.

Reply:
I've posted updated screenshot. I'll update the project screenshot if it looks good to you.